### PR TITLE
Build culture page

### PR DESCRIPTION
**Pull request by carkod**
## Done

Build culture a.k.a company-culture

## QA

- Check https://canonical-com-832.demos.haus/careers/company-culture with [design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=7794-19900&t=JpE73eNhcYSLeFv2-0)
- Compare with [copy doc](https://docs.google.com/document/d/1PAeP_ZO4JzSBCi-fRp6F7SYtSSRY-4DcWLUf9HJaMxs/edit#)

- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-178?atlOrigin=eyJpIjoiYWM4YTE2MWU2NjhmNGMwY2IyYTViNzdkZDgxM2I5YzMiLCJwIjoiaiJ9

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

File: templates/careers/company-culture.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
+{% extends '/careers/base-tabs.html' %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1PAeP_ZO4JzSBCi-fRp6F7SYtSSRY-4DcWLUf9HJaMxs/edit#heading=h.yys3mdcj4xua{% endblock meta_copydoc %}
+
+{% block title %}Company Culture | Careers{% endblock %}
+
+{% block meta_description %}Canonical prefers internal promotion and encourages high-performing colleagues to pursue diverse roles at the company over the course of their career, to develop a well-rounded perspective.{% endblock %}
```
Comment (anthonydillon) [inline templates/careers/company-culture.html:6]: This is not the description for this page

File: templates/careers/company-culture.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
+{% extends '/careers/base-tabs.html' %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1PAeP_ZO4JzSBCi-fRp6F7SYtSSRY-4DcWLUf9HJaMxs/edit#heading=h.yys3mdcj4xua{% endblock meta_copydoc %}
```
Comment (anthonydillon) [inline templates/careers/company-culture.html:2]: Can you remove the heading hash

File: static/sass/styles.scss [hunk 3]
```diff
@@ -437,3 +437,19 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
     background-size: inherit;
   }
 }
+
+.u-data-border--top {
+  border-top: 3px solid $color-dark;
+}
+
+.u-image-margin--top {
+  margin-top: 0.5625rem;
```
Comment (anthonydillon) [inline static/sass/styles.scss:446]: Your still using online styling in the page

File: templates/careers/company-culture.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+</div>
+<div class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom">
+    <div class="col-medium-2 col-3">
+      <h3 class="p-muted-heading">Fully distributed <br class="u-hide--small" />since 2004</h3>
+    </div>
+    <div class="col-medium-4 col-9">
+      <div class="row">
+        <div class="col-3 col-medium-1" style="margin-top: 0.5625rem">
```
Comment (anthonydillon) [inline templates/careers/company-culture.html:49]: I would suggest trying to get the profile images and quotes side by side in small screens.

File: templates/careers/company-culture.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+<div class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom">
+    <div class="col-medium-2 col-3">
+      <h3 class="p-muted-heading">Committed <br class="u-hide--small">
+        to excellence</h3>
+    </div>
+    <div class="col-medium-4 col-9">
+      <div class="p-strip is-shallow u-no-padding--top">
+        <p>We aspire to lead on the global stage. Most of us are avid students of greatness - we are interested in how the latest technologies work, how the best organisations work and how the smartest teams collaborate. We aim to hold our own in that crowd. We pursue engineering excellence and love to engineer excellence in all aspects of our work. From releasing the world’s most popular Linux reliably to developing our <a href="/documentation">documentation</a> practice.
```
Comment (anthonydillon) [inline templates/careers/company-culture.html:116]: Can you encode the dash here please?

Comment (anthonydillon) [reply templates/careers/company-culture.html:116]: I would suggest "documentation practise" should be the text of the link.

File: templates/careers/company-culture.html [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+<div class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom">
+    <div class="col-medium-3 col-3">
+      <h3 class="p-muted-heading">We’re Canonical</h3>
+    </div>
+    <div class="col-medium-6 col-9">
+      <div class="row">
+        <div class="col-medium-2 col-3">
+          <h4 class="p-heading--2 u-data-border--top"><strong>950+</strong><br>Employees</h3>
```
Comment (anthonydillon) [inline templates/careers/company-culture.html:28]: Starting h4, closing h3

File: templates/careers/company-culture.html [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+        </div>
+        <div class="col-medium-2 col-3">
+          <h4 class="p-heading--2 u-data-border--top"><strong>89</strong><br>Countries</h3>
```
Comment (anthonydillon) [inline templates/careers/company-culture.html:31]: Starting h4, closing h3

File: templates/careers/company-culture.html [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+        </div>
+        <div class="col-medium-2 col-3">
+          <h4 class="p-heading--2 u-data-border--top"><strong>78</strong><br>Languages</h3>
```
Comment (anthonydillon) [inline templates/careers/company-culture.html:34]: Starting h4, closing h3

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Thanks @carkod this page is looking good. Few suggestions in the code.

**Comment by carkod** [comment]
> @carlos looks great! A couple of small glitches:
> 
> * space under image making text misaligned:
> 

@lyubomir-popov Can I get another review? Thanks

**Comment by lyubomir-popov** [comment]
@carkod the grid nesting is a bit wrong:
image
that happens because you put stuff inside a row that is not a child of col-* - so the grid implicitly creates columns for those things. your markup should lookc like this:
image
and not like this (current): 
image
to add more stuff, you ca nwrap it in a col-9 or ad another row inside

**Comment by lyubomir-popov** [comment]
Adding plus one so you don't have to request another review but please fix the grid as described above

**Comment by marinacastejon** [comment]
Hi @carkod as discussed in MM, can you please change the "Remote" link so that it says "Travel and Lifestyle" in the meantime? Otherwise people click on sth that says Remote work and end up in a page that has a different name. I've created a task in JIRA to update it later on with a dependency on the updated remote work page

**Comment by carkod** [comment]
> @carkod the grid nesting is a bit wrong: image that happens because you put stuff inside a row that is not a child of col-* - so the grid implicitly creates columns for those things. your markup should lookc like this: ...

Got it. Done.

**Comment by carkod** [comment]
> Hi @carkod as discussed in MM, can you please change the "Remote" link so that it says "Travel and Lifestyle" in the meantime? Otherwise people click on sth that says Remote work and end up in a page that has a different name. I've created a task in JIRA to update it later on with a dependency on the updated remote work page

@marinacastejon can you take another look?

**Comment by marinacastejon** [comment]
Hi @carkod Thanks! I found a small thing, the link Explore careers, redirects to /careers, but it should go to the career explorer page. Let me know when you're done and I'll +1 this.

**Comment by marinacastejon** [comment]
👍 LGTM, thanks @carkod

**Pull request merged**